app/models/field.rb

```diff
@@ -0,0 +1,19 @@
+class Field < ApplicationRecord
+  belongs_to :calculator
+  before_create :set_selector
+  validates :uuid, :type, :label, :kind, presence: true
+  enum kind: { form: 0, parameter: 1, result: 2 }
+
+  private
+    def set_selector
+      if self.selector.present?
+        selected_rows_count = Field.where(kind: :kind).count
```

Comment:
`where(kind: :kind)` this will retrieve all records with `kind="kind"`. You do not need this

Reply:
I've not found in rails docs how to invoke current enum option. There are only examples how to check enum option which you already know.